fearless/Modules/Staking/YourValidators/View/YourValidatorTableCell.swift

```diff
@@ -29,6 +29,12 @@ class YourValidatorTableCell: UITableViewCell {
         return imageView
     }()
 
+    let errorImageView: UIImageView = {
```

Comment:
I suggest to use UIStackView so we don't have holes when some status is hidden

Reply:
Done